code/implementation/host-stats/src/main/java/io/cattle/platform/host/stats/api/HostStatsOutputFilter.java

```diff
@@ -1,11 +1,15 @@
 package io.cattle.platform.host.stats.api;
```

Comment:
The class really shouldn't be called `HostStatsOutputFilter` since it deals with all types of stats. Would `StatsOutputFilter` be more appropriate.